src/Module/Profile.php

```diff
@@ -54,15 +54,28 @@ public static function rawContent()
 	{
 		if (ActivityPub::isRequest()) {
 			$user = DBA::selectFirst('user', ['uid'], ['nickname' => self::$which]);
+			$data = [];
 			if (DBA::isResult($user)) {
 				$data = ActivityPub\Transmitter::getProfile($user['uid']);
+			}
+
+			if (!empty($data)) {
 				System::jsonExit($data, 'application/activity+json');
 			} elseif (DBA::exists('userd', ['username' => self::$which])) {
 				// Known deleted user
-				System::httpExit(410);
+				$data = [
+					'@context' => ActivityPub::CONTEXT,
+					'id' => System::baseUrl() . '/profile/' . self::$which,

```

Comment:
```suggestion
					'id' => $app->getBaseUrl() . '/profile/' . self::$which,
```

Reply:
I appreciate your feedback, but I almost committed WSOD-inducing code. If you make a suggestion through GitHub, may I ask you to check for actual feasability? In this case `$app` doesn't exist in this function, but the suggestion display doesn't allow for an easy check.

Reply:
I'm sorry, next time I'll just add a "real" suggestion in case I'm sure with it. I saw it as a suggestion to change it "like/at least" that way.

Reply:
I get it, and I've been on your end of this problem, where my suggestion in a code block (without even the convenient `Commit suggestion` button) has been used as is even though it was bogus (see https://github.com/friendica/friendica/commit/3de1fc3097cc7a1e38bf1a0c3cae5ae4ec8067eb).

Suggesting code is a tricky business, I'm tripping over myself regularly so the above ask is valid for myself as well.